lib/SymbolGraphGen/Symbol.h

```diff
@@ -69,6 +69,9 @@ class Symbol {
   void serializeGenericParam(const swift::GenericTypeParamType &Param,
                              llvm::json::OStream &OS) const;
 
+  void serializeParameterList(const swift::ParameterList *ParamList,
```

Comment:
I wanted for this to be an invisible change but if I didn't declare it here I got an error 
> Symbol.cpp:409:14: error: out-of-line definition of 'serializeParameterList' does not match any declaration in 'swift::symbolgraphgen::Symbol'

Is there a way to hide this API?

Reply:
If you don't want to add it to the class, you could write it as a static function inside the file (with extra parameters for not being a class method any more), or you could write it like a closure inside `serializeFunctionSignature`:

```cpp
auto serializeParameterList = [&](const swift::ParameterList *ParamList) {
  // ...
}
```

...on the other hand, this method is already considered class-private since it was declared here, so i don't consider it a big deal to have added it to the header like this.

Reply:
Thanks! That was exactly what I was looking for.